test/test_import_hook.py

```diff
@@ -74,49 +74,133 @@ def _test_import_hook(importhook_tempdir, typechecker):
     assert counter + 1 == jaxtyping._test_import_hook_counter
 
 
-def test_import_hook_typeguard_old(importhook_tempdir):
-    _test_import_hook(importhook_tempdir, ("typeguard", "typechecked"))
+# Tests start here...
```

Comment:
?

Reply:
Oops.. And the comment just helps to visually separate stuff that is not tests, and stuff that is tests.